src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -148,5 +287,30 @@ public void TestStartOnUnixWithBadFormat()
 
         [DllImport("libc")]
         private static extern int chmod(string path, int mode);
+
+        /// <summary>
+        /// Checks if the program is installed
+        /// </summary>
+        /// <param name="program"></param>
+        /// <returns></returns>
+        private bool IsProgramInstalled(string program)
```

Comment:
The logic of this method is copied from here: https://github.com/dotnet/corefx/blob/fb0a5a037989d120cc4f42371eb1274cfc291442/src/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L380-L393